### PR TITLE
feature/GAL-1360

### PR DESCRIPTION
**Pull request by fais-2**
## Description
This ticket is to amend all episode event updates to populate the relevant source system of the update e.g. GPS for 'Invited' episode events.
https://nhsd-jira.digital.nhs.uk/browse/GAL-1360

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: terraform/src/createEpisodeRecords/createEpisodeRecords.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ import {
 
 const client = new DynamoDBClient({ region: "eu-west-2" });
 const ENVIRONMENT = process.env.ENVIRONMENT;
-const UNSUCCESSFUL_RESPONSE = 400;
```
Comment (chenglawson1) [inline terraform/src/createEpisodeRecords/createEpisodeRecords.js:9]: Why is this removed, is it not used anymore?

Comment (fais-2) [reply terraform/src/createEpisodeRecords/createEpisodeRecords.js:9]: @chenglawson1 yes, not used anywhere

**Review by MichaelW-NHS — APPROVED** [review]
